web-client/src/presenter/actions/CourtIssuedDocketEntry/setCourtIssuedDocumentInitialDataAction.test.ts

```diff
@@ -108,4 +122,50 @@ describe('setCourtIssuedDocumentInitialDataAction', () => {
 
     expect(result.state.form).toEqual({});
   });
+
+  it('should set the judge name when eventcode is OJR and docketEntry was signed by the judge', async () => {
+    const result = await runAction(setCourtIssuedDocumentInitialDataAction, {
+      modules: {
+        presenter,
+      },
+      props: {
+        docketEntryId: docketEntryIds[2],
+      },
+      state: {
+        caseDetail: MOCK_CASE,
+        form: {},
+        judges: [
+          {
+            judgeFullName: 'John O. Colvin',
+            name: 'Colvin',
+            userId: '4497d22-0874-4c5e-ac98-a843d1472baf',
```

Comment:
Here and below, can we get this data from the `judgeColvin` user in `mockUsers.ts`?